### DISCUSSION TITLE
SyntaxError: Unexpected token < in JSON at position 0

### DISCUSSION THREAD

**Discussion by serggermob** [chriskyfung/AutoFetcher-IG-Stories-to-GDrive#47] [Q&A] [answered]
SyntaxError: Unexpected token < in JSON at position 0

How do I fix it? Thank you

**Comment by chriskyfung** [comment, marked as answer]
Duplicate of #45

**Comment by chriskyfung** [comment]
The apps script failed to parse response in JSON format. It can happen when the Instagram endpoint rejects a request and returns a HTML page to ask you re-login.

You can to open and login to your Instagram account using a web browser. Sometimes, Instagram may also ask you to verify your account. Then, lookup the new values of cookie and required parameters and update them to your Google Sheet.
